.github/workflows/master.yml

```diff
@@ -238,7 +238,10 @@ jobs:
       - uses: ./.github/actions/pipdeptree
       - name: Run tests
         run: |
-          pytest tests/evaluate
+          pytest tests/evaluate --ignore=tests/evaluate/test_default_evaluator_delta.py
+      - name: Run tests with delta
+        run: |
+          pytest tests/evaluate/test_default_evaluator_delta.py
```

Comment:
making this change since the spark sessions from `tests/evaluate/test_evaluation` and `tests/evaluate/test_default_evaluator` are not fully isolated. 

for `tests/evaluate/test_default_evaluator_delta.py` we need to test the case with no spark session available, and test with a spark session that has the delta extension.

Reply:
How are we testing this with both spark sessions and without it? We are ignoring it in the previous run?

Let me know if I am missing something?

Reply:
@sunishsheth2009 , `test_default_evaluator_delta` has a different fixture to create a spark session with delta. we have a test with and without the fixture.